packages/components/src/templates/next/components/complex/InfoCols/InfoCols.tsx

```diff
@@ -75,14 +107,32 @@ const InfoCols = ({
   title,
   subtitle,
   infoBoxes,
+  layout,
   site,
   LinkComponent,
 }: InfoColsProps) => {
+  const simplifiedLayout = layout === "homepage" ? "homepage" : "default"
```

Comment:
Should we extract this to a reusable function instead? So the layout prop can be changed in a single place if we ever want to add more layout types in the future